__tests__/news.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require('../app');
+
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+
+describe('GET /api/topics', () => {
+    test('should return correct status code', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    });
+    test('should return an array of topic objects with correct properties', () => {
+        return request(app)
+        .get('/api/topics')
+        .then((response) => {
+            const topicsArray = response.body.topic
+            topicsArray.forEach((topic) => {
```

Comment:
If you're going to iterate through to check each items properties, which is a great idea, we should also check the length of the array that comes back.
If an empty array was passed through here this test would pass incorrectly.